src/System.Management.Automation/engine/DataStoreAdapterProvider.cs

```diff
@@ -251,6 +251,14 @@ public override string ToString()
         /// </summary>
         public bool VolumeSeparatedByColon { get; internal set; } = true;
 
+        /// <summary>
+        /// Gets the path separators characters for this provider.
+        /// </summary>
+        /// <returns>
+        /// Array of strings representing the path separators characters for this provider.
```

Comment:
Comment needs to be updated. It's no longer a string array.

Reply:
Done